EventHeader header
uint64 counter